extensions/AbuseFilter/AbuseFilter.class.php

```diff
@@ -1080,10 +1080,11 @@ public static function takeConsequenceAction( $action, $parameters, $title,
 				// Mark with a tag on recentchanges.
 				global $wgUser;
 
-				$actionID = implode( '-', array(
-						$title->getPrefixedText(), $wgUser->getName(),
-							$vars->getVar( 'ACTION' )->toString()
-					) );
+				$UserIdOrIp =
```

Comment:
Minor thing, but shouldn't this be consistently named with the below: `$userIdOrIp`?

Reply:
yep, that's true
https://github.com/Wikia/app/pull/14999